python.d/traefik.chart.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
+# Description: traefik netdata python.d module
+# Author: Alexandre Menezes (@ale_menezes)
+
+from json import loads
+from collections import defaultdict
+from bases.FrameworkServices.UrlService import UrlService
+
+# default module values (can be overridden per job in `config`)
+update_every = 1
+priority = 60000
+retries = 10
+
+# charts order (can be overridden if you want less charts, or different order)
+ORDER = [
+    'response_statuses',
+    'response_codes',
+    'detailed_response_codes',
+    'requests',
+    'total_response_time',
+    'average_response_time',
+    'average_response_time_per_iteration',
+    'uptime'
+]
+
+CHARTS = {
+    'response_statuses': {
+        'options': [None, 'Response statuses', 'requests/s', 'responses', 'traefik.response_statuses', 'stacked'],
+        'lines': [
+            ['successful_requests', 'success', 'incremental'],
+            ['server_errors', 'error', 'incremental'],
+            ['redirects', 'redirect', 'incremental'],
+            ['bad_requests', 'bad', 'incremental'],
+            ['other_requests', 'other', 'incremental']
+        ]},
+    'response_codes': {
+        'options': [None, 'Responses by codes', 'requests/s', 'responses', 'traefik.response_codes', 'stacked'],
+        'lines': [
+            ['2xx', None, 'incremental'],
+            ['5xx', None, 'incremental'],
+            ['3xx', None, 'incremental'],
+            ['4xx', None, 'incremental'],
+            ['1xx', None, 'incremental'],
+            ['other', None, 'incremental']
+        ]},
+    'detailed_response_codes': {
+        'options': [None, 'Detailed response codes', 'requests/s', 'responses', 'traefik.detailed_response_codes', 'stacked'],
+        'lines': [
+        ]},
+    'requests': {
+        'options': [None, 'Requests', 'requests/s', 'requests', 'traefik.requests', 'line'],
+        'lines': [
+            ['total_count', 'requests', 'incremental']
+        ]},
+    'total_response_time': {
+        'options': [None, 'Total response time', 'seconds', 'timings', 'traefik.total_response_time', 'line'],
+        'lines': [
+            ['total_response_time_sec', None, 'absolute', 1, 1]
+        ]},
+    'average_response_time': {
+        'options': [None, 'Average response time', 'milliseconds', 'timings', 'traefik.average_response_time', 'line'],
+        'lines': [
+            ['average_response_time_sec', None, 'incremental', 1000000, 1000]
```

Comment:
`None` => `response`

Reply:
@l2isbad you're right. `average_response_time_sec` it's relative so in this case we the configuration would be:
`['average_response_time_sec', 'response', 'absolute', 1, 1000]`

To show average response time in milliseconds.